corehq/apps/builds/templates/builds/bootstrap5/edit_menu.html

```diff
@@ -15,103 +15,89 @@
-          <th></th>
-        </tr>
-        </thead>
-        <tbody data-bind="foreach: versions">
-        <tr>
-          <td>
-            <select class="form-control" data-bind="
-                                    options: $root.available_versions,
-                                    value: version
-                                ">
-            </select>
-          </td>
-          <td>
-            <input class="form-control" type="text" data-bind="value: badge">
-          </td>
-          <td>
-            <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only"/>
-          </td>
-          <td>
-            <button type="button" class="btn btn-outline-danger" data-bind="click: $root.removeVersion">
-              <i class="fa fa-remove"></i>
-            </button>
-          </td>
-        </tr>
-        </tbody>
-        <tr>
-          <td colspan="5">
-            <button data-bind="click: addVersion" class="btn btn-outline-primary">
-              <i class="fa fa-plus"></i>
-              {% trans "Add a Version" %}
-            </button>
-          </td>
-        </tr>
-      </table>
+    <div class="card-text">
+      <form id="menu-form" method="POST">{% csrf_token %}
+        <table id="menu-table" class="table table-striped">
+          <thead>
+          <tr>
+            <th>{% trans "Version" %}</th>
+            <th>{% trans "Label" %}</th>
+            <th>{% trans "Superuser Only" %}</th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody data-bind="foreach: versions">
+          <tr>
+            <td>
+              <select class="form-control" data-bind="
+                                      options: $root.available_versions,
+                                      value: version
+                                  ">
+              </select>
+            </td>
+            <td>
+              <input class="form-control" type="text" data-bind="value: label">
+            </td>
+            <td>
+              <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only"/>
+            </td>
+            <td>
+              <button type="button" class="btn btn-outline-danger" data-bind="click: $root.removeVersion">
+                <i class="fa fa-remove"></i>
+              </button>
+            </td>
+          </tr>
+          </tbody>
+        </table>
+        <button data-bind="click: addVersion" class="btn btn-outline-primary">
+          <i class="fa fa-plus"></i>
+          {% trans "Add a Version" %}
+        </button>
 
-      <hr/>
+        <hr/>
 
-      <fieldset>
-        <legend>Defaults</legend>
-        <div class="form-group">
-          <label class="form-label col-md-3" for="select-default-1">
-            {% trans "Default 1.x Build" %}
-          </label>
-          <div class="col-md-8">
+        <fieldset>
+          <legend>Defaults</legend>
+          <div class="mb-3">
```

Comment:
Going by https://www.commcarehq.org/styleguide/b5/organisms/forms/#html-forms I wonder if there should be a `row` class on this div.

Reply:
I see what you mean. I was going off the commentary in the tool ([this bit](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/utils/bootstrap/changes_guide/form-group.txt)) and using the [maintenance alerts page](https://staging.commcarehq.org/alerts/) as a model. Those imply a different style of form than what's in the style guide.

@biyeun can you weigh in here? What approach should we take for forms?

Reply:
i like moving away from horizontal forms where we can, as they are less usable than vertical forms. I should update the styleguide to reflect this.

Reply:
though, thinking about this...for the two column settings type forms, we should maintain the current horizontal form styling so that there isn't a huge jump between bootstrap3 and 5 page styles, then when we've fully moved over we can decide? I think for single paged layouts and modals (where horizontal space is limited) we should opt for vertical forms

Reply:
that said, i think the approach taken here to styling the form is fine

Reply:
okay, cool. I'll merge this one and keep those thoughts in mind for future migrations.

Reply:
Although if you lean towards vertical forms long term, I'd vote for doing it as part of the B5 migration where developers are able and willing to do so, I doubt it's going to be a major upset for users. I wouldn't necessarily change it for, say, Web Apps, because that's **so** high visibility, but for most random settings pages in HQ I think it's fine.